GoogleCloudPlatform/microservices-demo
# Set OTEL service Name in node js services

### Background 
The traces reported by the currency service shown the service name as `unknown` regardless of the value of the environment variable `OTEL_SERVICE_NAME`. 

### Fixes 
#2142

### Change Summary
Sets the service name in the `NodeTraceProvider`

### Additional Notes


### Testing Procedure
Deploy the application with the tracing enable and check for spans with the service name `currencyservice`
